packages/libs/error-stack/Makefile.toml

```diff
@@ -5,6 +5,15 @@ CARGO_CLIPPY_HACK_FLAGS = "--workspace --feature-powerset --exclude-features def
 CARGO_TEST_HACK_FLAGS = "--workspace --feature-powerset --exclude-features default --optional-deps eyre,anyhow"
 CARGO_MIRI_FLAGS = "--tests --lib"
 
+[tasks.test]
+env = { CARGO_TEST_HACK_FLAGS = "${CARGO_TEST_HACK_FLAGS} --group-features anyhow,eyre --group-features std,hooks --group-features spantrace,futures,small" }
```

Comment:
Currently reviewing https://github.com/hashintel/hash/pull/794, which requires a specific set of features for `cargo insta`. I think, that `std` and `hooks` shouldn't be grouped together. Generally, I think `std` shouldn't be grouped at all.

Reply:
complex-wise this shouldn't be a big impact as `hook` is implying `std`.